components/graph/Followers.tsx

```diff
@@ -0,0 +1,112 @@
+import { ActionButton } from '@/common/buttons'
+import { LoadMoreButton } from '@/common/LoadMoreButton'
+import { ConfirmationModal } from '@/common/modals/confirmation'
+import { AccountsGrid } from '@/repositories/AccountView'
+import { useLabelerAgent } from '@/shell/ConfigurationContext'
+import { useWorkspaceAddItemsMutation } from '@/workspace/hooks'
+import { useInfiniteQuery } from '@tanstack/react-query'
+import { useState } from 'react'
+import { toast } from 'react-toastify'
+
+export function Followers({ id, count }: { id: string; count?: number }) {
+  const [isConfirmationOpen, setIsConfirmationOpen] = useState(false)
+  const labelerAgent = useLabelerAgent()
+  const [isAdding, setIsAdding] = useState(false)
+  const { mutateAsync: addItemsToWorkspace } = useWorkspaceAddItemsMutation()
+  const { error, isLoading, data, fetchNextPage, hasNextPage } =
+    useInfiniteQuery({
+      queryKey: ['followers', { id }],
+      queryFn: async ({ pageParam }) => {
+        const { data } = await labelerAgent.api.app.bsky.graph.getFollowers({
```

Comment:
nit: `.api.` no longer required (deprecated)

```suggestion
        const { data } = await labelerAgent.app.bsky.graph.getFollowers({
```